qutip/solver/brmesolve.py

```diff
@@ -0,0 +1,212 @@
+"""
+This module provides solvers for the Lindblad master equation and von Neumann
+equation.
+"""
+
+__all__ = ['mesolve', 'MeSolver']
+
+import numpy as np
+import inspect
+from time import time
+from .. import Qobj, QobjEvo, coefficient, Coefficient
+from ..core.blochredfield import bloch_redfield_tensor, SpectraCoefficient
+from ..core.cy.coefficient import InterCoefficient
+from ..core import data as _data
+from .solver_base import Solver
+from .options import SolverOptions
+
+
+def brmesolve(H, psi0, tlist, a_ops=[], e_ops=[], c_ops=[],
+              args={}, sec_cutoff=0.1, options=None):
+    """
+    Solves for the dynamics of a system using the Bloch-Redfield master
+    equation, given an input Hamiltonian, Hermitian bath-coupling terms and
+    their associated spectrum functions, as well as possible Lindblad collapse
+    operators.
+
+    Parameters
+    ----------
+    H : Qobj / list
+        System Hamiltonian given as a Qobj or
+        nested list in string-based format.
```

Comment:
We should probably make the description for `H` more similar to the description of `H` in other solvers (i.e. now it can accept anything `QobjEvo` accepts, not just a Qobj or list.